src/test/java/com/google/sps/SentimentServletTest.java

```diff
@@ -0,0 +1,113 @@
+package com.google.sps;
+
+import com.google.api.gax.rpc.ApiException;
+import com.google.cloud.language.v1.Document;
+import com.google.cloud.language.v1.LanguageServiceClient;
+import com.google.cloud.language.v1.Sentiment;
+import com.google.sps.servlets.SentimentServlet;
+import java.io.BufferedReader;
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.StringReader;
+import java.io.StringWriter;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.Answers;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.Spy;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.mockito.ArgumentMatchers.*;
+import static org.mockito.Mockito.*;
+
+@RunWith(MockitoJUnitRunner.class)
+
+public final class SentimentServletTest {
+
+  private final Float SCORE = new Float(0.05f);
+  private StringWriter stringWriter;
+  private PrintWriter writer;
+
+  @Mock(answer = Answers.RETURNS_DEEP_STUBS) HttpServletRequest requestMock;
+  @Mock(answer = Answers.RETURNS_DEEP_STUBS) LanguageServiceClient languageServiceMock;
+  @Mock(answer = Answers.RETURNS_DEEP_STUBS) Sentiment sentimentMock;
+
+  @Spy HttpServletResponse responseSpy;
+
+  @InjectMocks SentimentServlet sentimentServlet;
+
+  @Before
+  public void setUp() throws IOException {
+    stringWriter = new StringWriter();
+    writer = new PrintWriter(stringWriter);
+    when(responseSpy.getWriter()).thenReturn(writer);
+  }
+
+  /**
+   * A Json String in the form of an array should be converted to a String of period separated
+   * comments. A sentiment score should be calculated and the score should be printed in a Json
+   * object. 
+   * Ex. "[foo, bar]" -> "{"commentScore":0.05}" 
+   */
+  @Test
+  public void jsonArrayPrintsVideoAnalysisJsonObject() throws IOException {
```

Comment:
Recommend adding another test with an array of 1 element. Just to make sure it works with the simple case as well. Remember to change the name of tests as well. `jsonSingleElementArray...` and `jsonMultiElementArray...`